lib/mailbox.c

```diff
@@ -27,6 +27,11 @@
 /* XXX Sample parameters, need to be tested for better performance. */
 #define GK_MEM_CACHE_SIZE (64)
 
+/*
+ * XXX Add ability to bulk enqueue/dequeue from a mailbox to speed
+ * up performance of transfer of IPv6 packets between blocks.
+ */
```

Comment:
This turned out to be unnecessary, as a *single* request (using one allocation from the LLS mailbox) can be used to hold an array of ND packets, achieving the same effect. The `submit_nd()` function changes to accommodate multiple ND packets in the latest patch, but the mailbox didn't need to change.